app/Http/Controllers/Backend/Support/MentionHelper.php

```diff
@@ -16,7 +17,8 @@ class MentionHelper
     private bool    $isCreating;
 
     public function __construct(Status $status, string $body = null) {
-        $this->status     = $status;
+        $this->status = $status;
+        $status->load('mentions', 'mentions.mentioned');
```

Comment:
```suggestion
        $status->load('mentions.mentioned');
```

`mentions` wird bereits von dem folgenden Attribut mitgeladen.